### PR TITLE
HTML preferring text/javascript is no longer a willful violation

### PR DESCRIPTION
**Pull request by mathiasbynens**
https://github.com/whatwg/html/pull/7938

### CODE REVIEW

File: files/en-us/web/http/basics_of_http/mime_types/common_types/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ IANA is the official registry of MIME media types and maintains a [list of all t
...
 | `.jpeg` `.jpg` | JPEG images                                                                                            | `image/jpeg`                                                                                                                                                                                                                                                                         |
-| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and its [reasoning](https://html.spec.whatwg.org/multipage/#dependencies:willful-violation), and [IETF](https://datatracker.ietf.org/doc/rfc9239/)) |
+| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and [IETF](https://datatracker.ietf.org/doc/rfc9239/)) |
```
Comment (sideshowbarker) [inline files/en-us/web/http/basics_of_http/mime_types/common_types/index.md:55]:
```suggestion
| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and [IETF](https://www.rfc-editor.org/rfc/rfc9239)) |
```

Comment (mathiasbynens) [reply files/en-us/web/http/basics_of_http/mime_types/common_types/index.md]: Thanks! I'm never quite sure which of these URLs is meant to be canonical. Wish the IETF would make it more clear.

Comment (sideshowbarker) [reply files/en-us/web/http/basics_of_http/mime_types/common_types/index.md]: Yeah the IETF has made it confusing… For MDN, we currently follow a practice of citing the https://www.rfc-editor.org URLs if there is one, and if not, then the https://datatracker.ietf.org URLs

File: files/en-us/web/http/basics_of_http/mime_types/common_types/index.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ IANA is the official registry of MIME media types and maintains a [list of all t
...
 | `.jpeg` `.jpg` | JPEG images                                                                                            | `image/jpeg`                                                                                                                                                                                                                                                                         |
-| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and its [reasoning](https://html.spec.whatwg.org/multipage/#dependencies:willful-violation), and [IETF](https://datatracker.ietf.org/doc/rfc9239/)) |
+| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and [IETF](https://www.rfc-editor.org/rfc/rfc9239)) |
```
Comment (sideshowbarker) [inline files/en-us/web/http/basics_of_http/mime_types/common_types/index.md:55]:
```suggestion
| `.js`          | JavaScript                                                                                             | `text/javascript` (Specifications: [HTML](https://html.spec.whatwg.org/multipage/#scriptingLanguages) and [RFC 9239](https://www.rfc-editor.org/rfc/rfc9239)) |
```

**Review by sideshowbarker — APPROVED** [review]
Super happy to see this reaching RFC. Thanks much for all your work on it

**Pull request merged**